CharacterCreation/07_Items.txt

```diff
@@ -2,164 +2,178 @@
+open a man-sized hole in 1 inch thick steel would take 10 full turns.
 
 Door Ram ($300): 
-Add 50 to a roll when attempting to kick open a door. Must have STR 8 to carry 
-or use.
+    Add 50 to a roll when attempting to kick open a door. Must have STR 8 to 
+carry or use.
 
 Bolt Cutters ($80):
-Used to cut through chain links, locks, cables, etc.
+    Used to cut through chain links, locks, cables, etc.
 
 Pneumatic Door Press ($700):
-Opens sliding, pneumatically controlled doors up to Militarized Level (airlock 
-specific doors exluded)
-
+    Opens sliding, pneumatically controlled doors up to Militarized Level 
+(airlock specific doors exluded)
 
 3D image builder, Static ($650):
-A 3m^3 framework that will build a rudimentary  3d image of whatever is inside
-it, exportable to a portable device.
+    A 3m^3 framework that will build a rudimentary  3d image of whatever is 
+inside it, exportable to a portable device.
 
 3D image builder, Portable ($800):
-Builds a 3D image of what its being pointed at, will need to see the object 
+    Builds a 3D image of what its being pointed at, will need to see the object 
 from multiple angles to capture the whole object. Up to 20m^3 of space.
 
 ==============================
 Wearables
 ==============================
 Night Vision Goggles ($300):
-IR visibility for low light conditions. Resolution clouds after 100m.
+    IR visibility for low light conditions. Resolution clouds after 100m.
 
 Gas Mask ($300):
-Prevents the inhalation of noxious or toxic air-borne compunds. Reduce PER 
+    Prevents the inhalation of noxious or toxic air-borne compunds. Reduce PER 
 based rolls (including strike rolls) by 10.
 
 ==============================
 Consumables
 ==============================
 
-MarkAll(tm) permanent marking pen. $2
-
-MediKit: $100
-For normal wounds (50% damage or less) roll percents under 50. On pass, heal 30
-+ percent rolled. For severe wounds (50% damage or more), roll percents under 
+Permanent Marker ($2):
+    MarkAll(tm) permanent marking pen of one of the following colors.
+
+* Black
+* Red
+* Blue
+* Dark Green
+* Orange
+* Purple
+* Gold
+* Silver
+
+MediKit ($100):
+    For normal wounds (50% damage or less) roll percents under 50. On pass, heal 
+30+ percent rolled. For severe wounds (50% damage or more), roll percents under 
 50. On pass, heal 30. If you have Skill in Medicine, see the skill sheet for 
 separate instructions.
 
-Repair kit: $100
-For normal damage (50% or less) roll percents under 50. On pass, repair 30 + 
+Repair kit ($100):
+    For normal damage (50% or less) roll percents under 50. On pass, repair 30 + 
 percent rolled. For severe wounds (50% or more), roll percents under 50. On 
 pass, repair 30.
-If you have Skill in Repair, see the skill sheet for separate instructions.
 
-Nanite Pack: $100
-Restores 2d10 + 20 nanites.
+    If you have Skill in Repair, see the skill sheet for separate instructions.
```

Comment:
Nonblocking inconsistency: Repair kits puts this line in a new paragraph but medkits do not.

Reply:
will change